tools/lib/h5diff.c

```diff
@@ -1485,9 +1485,6 @@ diff_match(hid_t file1_id, const char *grp1, trav_info_t *info1, hid_t file2_id,
                 } /* end else */
             }     /* end while */
 
-            for (i = 1; (int)i < g_nTasks; i++)
-                MPI_Send(NULL, 0, MPI_BYTE, (int)i, MPI_TAG_END, MPI_COMM_WORLD);
```

Comment:
This same loop already exists in `phdiff_dismiss_workers` in `tools/lib/h5diff.c` and gets called by the manager rank on ph5diff exit. This loop was just causing the same messages to be sent twice and end up unmatched.